airflow/gcp/operators/dlp.py

```diff
@@ -1032,7 +1032,7 @@ def execute(self, context):
         )
 
 
-class CloudDLPGetJobTripperOperator(BaseOperator):
+class CloudDLPGetJobTriggerOperator(BaseOperator):
```

Comment:
```suggestion
class CloudDLPGetDLPJobTriggerOperator(BaseOperator):
```
Let's be consistent. @mik-laj do you think we need to repeat DLP two times in the name?

Reply:
I think, we should consistent.  If the API uses such a convention, then we should also use it. If we want to delete the DLP shortcut, we should first change the API.

Reply:
Done